### DISCUSSION TITLE
性能有点低下，控件不太好用

### DISCUSSION THREAD

**Discussion by shanyan-wcx** [Richasy/Bili.Uwp#585] [想法]
用起来卡卡的，页面加载比较慢，滚动卡得不行。
播放控件用起来很别扭，能不能改成网页端那种，不要把弹幕和控件分两层。

**Comment by hez2010** [comment]
可能是因为没有开启硬件加速或者硬件配置带不起应用中使用的大量动画和显示效果

**Comment by shanyan-wcx** [comment]
硬件配置肯定没问题，硬件加速的选项在应用设置里没找到啊

**Reply by hez2010** [reply]
那这个可能显卡或者驱动之类的有关，已知屏蔽核显只用独显可能会导致动画和视频降到不到20帧

**Reply by X0980** [reply]
> 那这个可能显卡或者驱动之类的有关，已知屏蔽核显只用独显可能会导致动画和视频降到不到20帧

我的电脑没有核显怎么办？首页刷起来确实卡

**Reply by hez2010** [reply]
好像和 nvidia freesync/g-sync 有关，尝试关闭一下看看。
